sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -298,6 +305,7 @@ export class ReceiverImpl<ReceivedMessageT extends ReceivedMessage | ReceivedMes
     const receivedMessages = await this._context.batchingReceiver.receive(
       maxMessageCount,
       options?.maxWaitTimeSeconds
+      // this._receiverOptions - No need to pass?
```

Comment:
Good point on that. I noticed there are a few places where things like this aren't plumbed. Let's file an issue - we should get to this and make sure it's all consistent.

(getMessageIterator() was another spot)

Reply:
Why do we say that we do not need to pass the options?

Reply:
Am ok with doing retries for receive operations in a separate PR and have this one focus only on operations for sending and the ones on the $management link

Reply:
> Why do we say that we do not need to pass the options?
Am ok with doing retries for receive operations in a separate PR and have this one focus only on operations for sending and the ones on the $management link

`maxWaitTimeSeconds` impacts this method and the timeout from `retryOptions` is in a direct conflict. We might need to agree on some sort of design before moving forward.

Reply:
Since we are dealing with retry on receive operations in a separate PR, please replace the comment here with a TODO statement instead of a comment that hints that we dont need to pass retry options

Reply:
Oops, missed this comment.